backend/cfg/llvmize.ml

```diff
@@ -274,14 +303,19 @@ module F = struct
 
   (* == LLVM instructions == *)
 
-  (* CR yusumez: loads/stores might be of different sizes *)
-  let ins_load t ident (reg : Reg.t) =
-    ins t "%a = load %a, ptr %a" pp_ident ident pp_machtyp_component reg.typ
-      (pp_reg_ident t) reg
+  let ins_load t ~src ~dst typ =
+    ins t "%a = load %a, ptr %a" pp_ident dst Llvm_typ.pp_t typ pp_ident src
+
+  let ins_store t ~src ~dst typ =
+    ins t "store %a %a, ptr %a" Llvm_typ.pp_t typ pp_ident src pp_ident dst
 
-  let ins_store t ident (reg : Reg.t) =
-    ins t "store %a %a, ptr %a" pp_machtyp_component reg.typ pp_ident ident
-      (pp_reg_ident t) reg
+  let ins_load_reg t ident (reg : Reg.t) =
+    ins_load t ~src:(get_ident_for_reg t reg) ~dst:ident
+      (Llvm_typ.of_machtyp_component reg.typ)
+
+  let ins_store_reg t ident (reg : Reg.t) =
```

Comment:
It'd be nice to have labeled `~src` and `~dst` args for the other store/load functions, to avoid confusion.